doc/topics/development/contributing.rst

```diff
@@ -219,201 +219,34 @@ Fork a Repo Guide_>`_ and is well worth reading.
-Git-level when those commits are merged-forward. In other words, only the
-commits are merged-forward and not the pull request text.
+This is very different from past releases which was 'year, month, dot release'.
+For example 2019.2 and 2019.2.3.
 
-.. _backporting-pull-requests:
-
-Backporting Pull Requests
-=========================
-
-If a bug is fixed on ``|repo_primary_branch|`` and the bug is also present on a
-currently-supported release branch, it will need to be back-ported to an
-applicable branch.
-
-.. note:: Most Salt contributors can skip these instructions
-
-    These instructions do not need to be read in order to contribute to the
-    Salt project! The SaltStack team will back-port fixes on behalf of
-    contributors in order to keep the contribution process easy.
-
-    These instructions are intended for frequent Salt contributors, advanced
-    Git users, SaltStack employees, or independent souls who wish to back-port
-    changes themselves.
-
-It is often easiest to fix a bug on the oldest supported release branch and
-then merge that branch forward into ``|repo_primary_branch|`` (as described earlier in this
-document). When that is not possible the fix must be back-ported, or copied,
-into any other affected branches.
-
-These steps assume a pull request ``#1234`` has been merged into ``|repo_primary_branch|``.
-And ``upstream`` is the name of the remote pointing to the main Salt repo.
-
-#.  Identify the oldest supported release branch that is affected by the bug.
-
-#.  Create a new branch for the back-port by reusing the same branch from the
-    original pull request.
-
-    Name the branch ``bp-<NNNN>`` and use the number of the original pull
-    request.
-
-    .. code-block:: bash
-
-        git fetch upstream refs/pull/1234/head:bp-1234
-        git checkout bp-1234
-
-#.  Find the parent commit of the original pull request.
-
-    The parent commit of the original pull request must be known in order to
-    rebase onto a release branch. The easiest way to find this is on GitHub.
-
-    Open the original pull request on GitHub and find the first commit in the
-    list of commits. Select and copy the SHA for that commit. The parent of
-    that commit can be specified by appending ``~1`` to the end.
-
-#.  Rebase the new branch on top of the release branch.
-
-    * ``<release-branch>`` is the branch identified in step #1.
-
-    * ``<orig-base>`` is the SHA identified in step #3 -- don't forget to add
-      ``~1`` to the end!
-
-    .. code-block:: bash
-
-        git rebase --onto <release-branch> <orig-base> bp-1234
-
-    Note, release branches prior to ``2016.11`` will not be able to make use of
-    rebase and must use cherry-picking instead.
-
-#.  Push the back-port branch to GitHub and open a new pull request.
-
-    Opening a pull request for the back-port allows for the test suite and
-    normal code-review process.
-
-    .. code-block:: bash
+Handling CVE’s
+--------------
 
```

Comment:
I've been looking up rules on whether you should put an 's to pluralize an acronym and it looks like it's up for debate with some grammar nerds. However, I would recommend changing it to "Handling CVEs" instead. The best proof I can offer is this entry in the Wiktionary: https://en.wiktionary.org/wiki/CVEs

Reply:
How about https://cve.mitre.org/cve/identifiers/index.html 🙃 (Agrees with "CVEs", though apparently CVE Entries is the most correct term)